src/main/java/apoc/couchbase/Couchbase.java

```diff
@@ -438,4 +440,18 @@
     }
     return result;
   }
+
+  private CouchbaseConnection getCouchbaseConnection(List<String> nodes, String bucket){
+    try (CouchbaseConnection couchbaseConnection = CouchbaseManager.getConnection(nodes, bucket)) {
+      return couchbaseConnection;
+    } catch (NoClassDefFoundError e){
+      throw new MissingDependencyException("Cannot find the jar into the plugins folder. \n"+
+              "Please put these jar in the plugins folder : \n\n" +
+              "java-client-2.3.1.jar\n" +
```

Comment:
probably leave off the exact versions? and just do -.x.y.z and link to the docs.